components/header

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Header } from 'react-native-elements';
+import { Icon } from 'react-native-vector-icons/FontAwesome';
+
+export default class AppHeader extends Component {
+    render() {
+        return (
+
+            <Header
+            leftComponent={{ icon: 'chevron-left', onPress: () => console.log('pressed'), color: '#000', size: 40 }}
+            centerComponent={{ text: 'header', style: { color: '#000', fontSize: 30, fontWeight: '500' } }}
+            containerStyle={{
+              backgroundColor: '#fff',
+            }}
```

Comment:
Format wise, can you remove the extra spaces inside of `return ();`, indent the contents of `<Header />`, use capitals when referring to colours `#FFF` and put the `containerStyle` on one line please.

Reply:
Sorry, my bad! I've fixed it and submitted another pull request.